src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoTreeNodes.cs

```diff
@@ -680,6 +680,87 @@ internal partial class DatabaseChildFactory : SmoChildFactoryBase
+                    Type = typeof(string),
+                    FilterType = FilterType.CONTAINS,
+                    IsNotFilter = true,
+                    Values = new List<object> { "db_accessadmin" },
+                });
+                filters.Add(new NodePropertyFilter
+                {
+                    Property = "Name",
+                    Type = typeof(string),
+                    FilterType = FilterType.CONTAINS,
+                    IsNotFilter = true,
+                    Values = new List<object> { "db_backupoperator" },
+                });
+                filters.Add(new NodePropertyFilter
+                {
+                    Property = "Name",
+                    Type = typeof(string),
+                    FilterType = FilterType.CONTAINS,
+                    IsNotFilter = true,
+                    Values = new List<object> { "db_datareader" },
+                });
+                filters.Add(new NodePropertyFilter
+                {
+                    Property = "Name",
+                    Type = typeof(string),
+                    FilterType = FilterType.CONTAINS,
+                    IsNotFilter = true,
+                    Values = new List<object> { "db_datawriter" },
+                });
+                filters.Add(new NodePropertyFilter
+                {
+                    Property = "Name",
+                    Type = typeof(string),
+                    FilterType = FilterType.CONTAINS,
+                    IsNotFilter = true,
+                    Values = new List<object> { "db_ddladmin" },
+                });
+                filters.Add(new NodePropertyFilter
+                {
+                    Property = "Name",
+                    Type = typeof(string),
+                    FilterType = FilterType.CONTAINS,
+                    IsNotFilter = true,
+                    Values = new List<object> { "db_denydatareader" },
+                });
+                filters.Add(new NodePropertyFilter
+                {
+                    Property = "Name",
+                    Type = typeof(string),
+                    FilterType = FilterType.CONTAINS,
+                    IsNotFilter = true,
+                    Values = new List<object> { "db_denydatawriter" },
+                });
+                filters.Add(new NodePropertyFilter
+                {
+                    Property = "Name",
+                    Type = typeof(string),
+                    FilterType = FilterType.CONTAINS,
+                    IsNotFilter = true,
+                    Values = new List<object> { "db_owner" },
+                });
+                filters.Add(new NodePropertyFilter
+                {
+                    Property = "Name",
+                    Type = typeof(string),
+                    FilterType = FilterType.CONTAINS,
+                    IsNotFilter = true,
+                    Values = new List<object> { "db_securityadmin" },
+                });
+                return filters;
```

Comment:
Is there a way to assemble the list of filters outside the getter and just return it immediately without needing to assemble every time?